ksql-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java

```diff
@@ -111,25 +112,31 @@ public void shouldAggregateWithNoWindow() throws Exception {
         KafkaTopicClient.TopicCleanupPolicy.COMPACT));
   }
 
+  @Test
+  public void shouldAggregateCustomUdaf() throws Exception {
```

Comment:
Can we incorporate the UDAFs into the `QueryTranslationTest`? If so, it would be a cheper way of doing the same thing.

Reply:
yeah - we can.